server/api/reviews.js

```diff
@@ -0,0 +1,26 @@
+const {Review} = require('../db/models/reviews')
+const router = require('express').Router()
+module.exports = router
+
+router.get('/', async function(req, res, next) {
+  try {
+    const reviews = await Review.findAll()
+    res.json(reviews)
+  } catch (err) {
+    next(err)
+  }
+})
+
+router.get('/:id', async function(req, res, next) {
+  try {
+    const review = await Review.findById(req.params.id)
+    res.json(review)
```

Comment:
Thanks Frank!! Looks good to me - I would add include: [{all:true}] as below to make sure all the information of the associations gets loader as well in this case users etc. Example below:

    const hats = await Hat.findAll({
      include: [{all: true}]
    })

:) - I would do this for the general get request and the id one as well.